history/characters/100000_vrykul.txt

```diff
@@ -6150,4 +6150,16 @@
 	father=102053	#Hoes
 	639.1.21={ birth=yes }
 	802.9.11={ death=yes }
+}
+102059={
+	name=Sif
+	female=yes
+	dna="abileddgcda"
+	culture=hyldnir religion=mystery_of_the_makers
+	martial=4 diplomacy=8 stewardship=7 intrigue=6 learning=5
+	add_trait=flamboyant_schemer add_trait=greedy add_trait=kind add_trait=patient add_trait=lustful
+	100.1.1={ birth=yes add_trait=creature_vrykul immortal_age=18 effect = { dynasty=none } }
```

Comment:
`effect = { dynasty=none }`? What is the point?

Reply:
so shes lowborn

Reply:
> so shes lowborn

@sejtalk she's lowborn even without it.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/78930611-72af9400-7ab5-11ea-8899-77595765d2d9.png)

</details>